diff -ur a/utils/tss.c b/utils/tss.c
--- a/utils/tss.c	2016-08-09 13:28:08.000000000 -0700
+++ b/utils/tss.c	2017-04-04 13:05:46.517591718 -0700
@@ -2265,7 +2265,7 @@
     }
     if (rc == 0) {
 	encryptParamBuffer = malloc(paramSize);
-	if (mask == NULL) {
+	if (encryptParamBuffer == NULL) {
 	    if (tssVerbose) printf("TSS_Command_DecryptXor: malloc %u failed\n",
 				   (unsigned int)sizeof(paramSize));
 	    rc = TSS_RC_OUT_OF_MEMORY;
@@ -2533,7 +2533,7 @@
     }
     if (rc == 0) {
 	decryptParamBuffer = malloc(paramSize);
-	if (mask == NULL) {
+	if (decryptParamBuffer == NULL) {
 	    if (tssVerbose) printf("TSS_Response_EncryptXor: malloc %u failed\n",
 				   (unsigned int)sizeof(paramSize));
 	    rc = TSS_RC_OUT_OF_MEMORY;
