diff -ur a/src/Unmarshal.c b/src/Unmarshal.c
--- a/src/Unmarshal.c	2016-08-09 13:28:07.000000000 -0700
+++ b/src/Unmarshal.c	2017-04-04 14:45:34.143514650 -0700
@@ -2128,7 +2128,6 @@
 {
     TPM_RC rc = TPM_RC_SUCCESS;
 
-    return rc;
     if (rc == TPM_RC_SUCCESS) {
 	rc = TPM2B_NAME_Unmarshal(&target->indexName, buffer, size);
     }
@@ -2138,6 +2137,7 @@
     if (rc == TPM_RC_SUCCESS) {
 	rc = TPM2B_MAX_NV_BUFFER_Unmarshal(&target->nvContents, buffer, size);
     }
+    return rc;
 }
 
 /* Table 118 - Definition of (TPM_ST) TPMI_ST_ATTEST Type <OUT> */
